packages/components/src/search/autocompleters/taxes.js

```diff
@@ -13,6 +13,7 @@ import { stringifyQuery } from '@woocommerce/navigation';
  * Internal dependencies
  */
 import { computeSuggestionMatch } from './utils';
+import { getTaxCode } from 'analytics/report/taxes/utils';
```

Comment:
Can you pull this function to a new package, or copy it into `packages/components`? This file won't exist in other apps using the components.

Reply:
Thanks for catching this, @ryelle!  I think @timmyc copied this into packages to fix up the build - https://github.com/woocommerce/wc-admin/blob/2985730ca5a541fbf05a2deb58781b4ddc0455b8/packages/components/src/search/autocompleters/utils.js#L28